src/errors.rs

```diff
@@ -10,10 +10,14 @@ mod errors_test;
 use fsio::error::FsIOError;
 use std::fmt;
 use std::fmt::Display;
+use std::error::Error;
+
+/// Envmt Library Result
+pub type EnvmntResult<T> = Result<T, EnvmntError>;
```

Comment:
can we move this one to the types.rs?

Reply:
Does it make sense though? It is inherently an alias of the error handling in the lib. Removing it towards a more generic "types.rs" is IMO adding semantic indirection.

Reply:
not 100% sure what it means, but feels a bit strange for people to import error:result and not types:result. maybe its just me, but feels more correct.

Reply:
Resulting is inherent to error handling. But your house your rules. I'm not particularly hung up about it. Moved it to types.